main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -205,12 +208,9 @@ object Lexer {
       start = new Position(sc.getLine, sc.getColumn, sc.getOffset)
     }
 
-    /**
-      * The sequence of tokens produced by the lexer.
-      *
-      * Note: The initial size of the array buffer has been determined by careful profiling.
-      */
-    val tokens: mutable.ArrayBuffer[Token] = new mutable.ArrayBuffer(initialSize = 256)
+    /** The sequence of tokens produced by the lexer. */
```

Comment:
The previous comment explained how the number was obtained.

I think you should explain here to. You might argue the code explains, but I think its to intricate.

Reply:
I gave it a go